src/index.css

```diff
@@ -1,13 +1,42 @@
-:root {
+/* ------- Style resets and custom properties ------- */
+@font-face {
+  font-family: 'FindingNemo';
+  src: url('/LostFish-5DOz.ttf') format('truetype');
+}
```

Comment:
Such a cute font!

Reply:
As Jenny said! It fits so well with Dory.